homeassistant/components/wled/config_flow.py

```diff
@@ -0,0 +1,125 @@
+"""Config flow to configure the AdGuard Home integration."""
+import logging
+from typing import Any, Dict, Optional
+
+import voluptuous as vol
+from wled import WLED, WLEDConnectionError
+
+from homeassistant.components.wled.const import DOMAIN  # pylint: disable=W0611
```

Comment:
Why not `from .const import DOMAIN`?

Reply:
Because I personally do not like relative imports. I think hey can become messy quite easy and are less readable compared to absolute ones. Absolute is also more explicit and will break if things get misplaced.

I haven't found any reference on a requirement on this, within Home Assistant, and is used both in the active codebase.

Reply:
Intra package imports should be relative to be more compatible with custom component overrides.

Reply:
@MartinHjelmare Ah, yes, good reasoning. Will adjust!